libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/Trace.hs

```diff
@@ -370,28 +404,36 @@ instance
          in trace
               (raiseMockError lastSlot nextSlotNo epochstate pdfs txsl gs)
               (error "FAILS")
-      -- Left pdfs -> trace ("Discard\n"++show(ppList (ppMockChainFailure reify) pdfs)) discard -- TODO should we enable this?
       Right mcs2 -> seq mcs2 (pure mockblock)
 
-  shrinkSignal (MockBlock _ _ xs) | SS.null xs = []
-  shrinkSignal (MockBlock i s xs) = [MockBlock i s (SS.drop 1 xs), MockBlock i s (SS.take (SS.length xs - 1) xs)]
+  shrinkSignal _ = []
 
-mapProportion :: (v -> Int) -> Map.Map k v -> Gen k
-mapProportion toInt m =
+--  shrinkSignal (MockBlock _ _ xs) | SS.null xs = []
+--  shrinkSignal (MockBlock i s xs) = [MockBlock i s (SS.drop 1 xs), MockBlock i s (SS.take (SS.length xs - 1) xs)]
```

Comment:
Is this dead code? Or we are planning on fixing shrinking and this can be useful? If so adding a TODO is a good idea, so we know what was the intent of this thing being left behind
```suggestion
```